src/internal/components/dataset/NamedSetList.tsx

```diff
@@ -57,6 +59,7 @@ export function NamedSetList({ headerIcon, headerText, value, status, onOpen }:
       setNamedSets((sets) => {
         const updatedSets = sets.map((set) => (set === namedSet ? editedSet : set));
         return sortNamedSetsAlphabetically(updatedSets);
+        onEditNamedSet?.(namedSet);
```

Comment:
Do not call this function after `return`.

Reply:
Oops